Doc/whatsnew/3.12.rst

```diff
@@ -62,91 +62,171 @@ Summary -- Release highlights
 .. This section singles out the most important changes in Python 3.12.
    Brevity is key.
 
+Python 3.12 is a medium-sized release with a mix of changes
```

Comment:
How do we measure the size of releases? Features, PEPs, commits? I'd probably just leave out the size of the release.

Reply:
It didn't feel small or big! The primary factor though was that the sentence felt empty otherwise "Python 3.12 has a mix of changes to the language and the standard library."

Perhaps: "Python 3.12 is the latest stable release of the Python programming language with a mix of changes to the language and the standard library."?

```suggestion
Python 3.12 is the latest stable release of the Python programming language with a mix of changes
```

Reply:
Sounds good!